test/test_repeat.js

```diff
@@ -644,7 +644,7 @@ describe('repeat', function() {
       );
   });
 
-  it.only('should emit a waiting event when adding a repeatable job to the waiting list', function(done) {
+  it('should emit a waiting event when adding a repeatable job to the waiting list', function(done) {
```

Comment:
Perhaps this was supposed to be a `.skip` since this test is failing?

Reply:
must have been committed while trying to fix the reason for the test to fail